docs/openapi.yaml

```diff
@@ -510,19 +510,146 @@ components:
   schemas:
     Publication:
       type: object
+      description: Complete Publication entity
```

Comment:
This description needs to be same as in nva-search-api. Currently its "description: Structure returned from successful seach". I suggest also creating a PR for nva-search-api